wordpress-mobile/gutenberg-mobile
# Fix store crash with gutenberg master

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1332

This PR brings us back to targeting gutenberg `master`

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/17228

To test: 
- Check that tests are all green
- Try to boot the demo app and perform regular actions
- Try to save and reload a post in particular

Update release notes:
